src/applications/vaos/components/ConfirmationDirectScheduleInfo.jsx

```diff
@@ -38,7 +38,7 @@ export default function ConfirmationDirectScheduleInfo({
           VA Appointment
         </div>
         <h2 className="vaos-appts__date-time vads-u-font-size--lg vads-u-margin-x--0">
-          {momentDate.format('MMMM D, YYYY [at] hh:mm a')}
+          {momentDate.format('MMMM D, YYYY [at] h:mm a')}
```

Comment:
I did a sweep looking for the `hh` format and replaced a couple instances, because I don't think it's every what we want. We either want `h` (single digit, 12 hour clock) or `HH` (two digit 24 hour clock)